orchagent/port/portschema.h

```diff
@@ -80,3 +85,5 @@
 #define PORT_ROLE                "role"
 #define PORT_ADMIN_STATUS        "admin_status"
 #define PORT_DESCRIPTION         "description"
+#define PORT_PT_INTF_ID            "pt_interface_id"
```

Comment:
can you update these new fields in doc/swss-schema.md

Reply:
@kperumalbfn I added the two fields to the documentation: https://github.com/sonic-net/sonic-swss/pull/2903/commits/c9ff12e76f4c764398b1e79a33d435ce8f0393a1